code/modules/atmospherics/gasmixtures/gas_mixture.dm

```diff
@@ -354,17 +352,15 @@ GLOBAL_LIST_INIT(gaslist_cache, init_gaslist_cache())
 	if(length(cached_gases ^ sharer_gases)) //if all gases were present in both mixtures, we know that no gases are 0
 		garbage_collect(cached_gases - sharer_gases) //any gases the sharer had, we are guaranteed to have. gases that it didn't have we are not.
 		sharer.garbage_collect(sharer_gases - cached_gases) //the reverse is equally true
-	sharer.after_share(src, atmos_adjacent_turfs)
+	if (initial(sharer.gc_share))

```

Comment:
is initial faster than a normal lookup?

Reply:
it is for a var that has not changed, like this one